configs/angular-best-practices.js

```diff
@@ -1,25 +1,25 @@
 module.exports = {
   plugins: ["@angular-eslint"],
+
   overrides: [
     {
       files: ["*.html"],
       plugins: ["@angular-eslint/template"],
       rules: {
-        "@angular-eslint/template/banana-in-box": "error",
         "@angular-eslint/template/button-has-type": "warn",
         "@angular-eslint/template/conditional-complexity": "warn",
         "@angular-eslint/template/cyclomatic-complexity": "warn",
+        "@angular-eslint/template/no-negated-async": "warn",
+        "@angular-eslint/template/use-track-by-function": "warn",
+
```

Comment:
extra newline

Reply:
I split the Off, Warn, and Error rules, with a newline between them.